substract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
Hi, there is an error in this code.
```suggestion
    return x - y
```